build/dependencies.yaml

```diff
@@ -168,6 +168,10 @@ dependencies:
       match: validTmplOut\s+=
     - path: cmd/kubeadm/app/util/template_test.go
       match: doNothing\s+=
+
+  - name: "k8s.gcr.io/pause: dependents"
```

Comment:
I believe this needs to be moved as well to `registry.k8s.io/pause`

Reply:
And also one above that declares dependencies for the Makefile only.

This is how it looks in master now: https://github.com/kubernetes/kubernetes/blob/819cadca66a7dccfa5c0772605fc90c667555c97/build/dependencies.yaml#L147-L154

Reply:
Maybe it's best to approve this one: https://github.com/kubernetes/kubernetes/pull/113403/files that covers it

Reply:
Maybe it seems better to use a separate PR to make these changes.